docs/Actions.md

```diff
@@ -23,6 +23,7 @@ import './path/to/other/Fastfile'
 - [Using git](#using-git)
 - [Using mercurial](#using-mercurial)
 - [Notifications](#notifications)
+- [Cordova](#cordova)
```

Comment:
yes sorry, i'm thinking of build a few more actions for cordova project, and I rushed.
When that happens I will add the section again if it makes sense